calico-cloud/release-notes/index.mdx

```diff
@@ -14,21 +14,33 @@ Calico Cloud includes new and improved Dashboards that enable operators to defin
 
 Calico Cloud includes a new UI that can be used to manage and configure global threat feeds
 
-## Namespace-based policy recommendations
+For more information, see [Manager UI tutorial](../tutorials/calico-cloud-features/tour.mdx#dashboard).
+
+### Namespace-based policy recommendations
 
 Calico Cloud has improved its policy recommendation engine to add namespace-based recommendations.
 This enables operators to easily implement microsegmentation for namespaces.
 
+For more information, see [Create policy recommendation](.../network-policy/generate-policy-recommendation.mdx).
```

Comment:
This link is not working in the preview output.

Reply:
Fixed, thanks!